resources/views/demo.blade.php

```diff
@@ -51,6 +51,19 @@
                 <code>{{ '<br>' }}</code>s are meant for single line breaks.
             </p>
         </x-section>
+
+        <x-section>
+            <h3>Feature Flags</h3>
+            <div class="flex justify-between">
+                <p>Beaten Games Player-facing UX</p>
+                @hasfeature("beat")
+                    Enabled
+                @else
+                    Disabled
+                @endhasfeature
+            </div>
+        </x-section>
```

Comment:
I didn't know if this was really the best place for this or not. Maybe the admin panel is better?

Reply:
It's nice to have it in the demo page for reference :)
For inspection by moderators the admin panel would be better, yeah.

Reply:
In https://github.com/RetroAchievements/RAWeb/pull/1811/commits/f01048c7e7fee83cdee66b77188a020485e082cc, I've broken out a `<x-feature-flags />` component that is being rendered on the demo page and admin.php.